src/CMakeLists.txt

```diff
@@ -56,12 +56,35 @@ if(BUILD_POLYHEDRAL_GRAVITY_PYTHON_INTERFACE)
 
     target_include_directories(polyhedral_gravity PUBLIC
             "${CMAKE_CURRENT_SOURCE_DIR}"
-            )
+    )
 
     target_link_libraries(polyhedral_gravity PUBLIC
             spdlog::spdlog
             tetgen
             xsimd
             Thrust
-            )
-endif()
\ No newline at end of file
+    )
+endif ()
+
+#####################################################
+# Building the Polyhedral Time Measurement Executable
+#####################################################
+if (BUILD_POLYHEDRAL_GRAVITY_TIME_MEASUREMENT)
+    message(STATUS "Building the Polyhedral Gravity Executable")
+
+    include(gtest)
```

Comment:
Why does this require `gtest`?

Reply:
Google benchmark internally uses gtest and ships with a bundled version. To avoid duplication I wanted the use the projects own version instead.

Reply:
I see the issue. In that case, the preferred solution would simply be to hinder Google Benchmark from fetching/ using its bundled Google Test.

You can achieve this by adding into the `bechmark.cmake` something like

```cmake
set(BENCHMARK_ENABLE_TESTING OFF CACHE BOOL "Disable benchmark testing" FORCE)
```

Reply:
This way, the option is **forced** to be always off, independent of what was set before.